bijlage02_GebruikGithub.md

```diff
@@ -19,6 +19,24 @@ Voorbeeld: een wijzigingsverzoek voor het aanpassen van de architectuurbeschrijv
 
 De _develop_ branch wordt dus niet gebruikt om wijzigingen op het document te maken maar dient als verzamelbranch voor de verschillende wijzigingen die in een volgende release moeten komen. _Patches_ kunnen wel direct op de _develop_ branch worden doorgevoerd.
 
+### Commits
+
+Commit messages zijn voor maintainers en derden belangrijk om een (historisch) beeld te krijgen van de evolutie van een standaard.
+Om het onderscheid duidelijk te maken tussen normatieve en niet-normatieve wijzigingen, moeten commits met enkel niet-normatieve wijzigingen starten met _Editorial:_.
```

Comment:
Ben het in principe een met commits in NL (wordt wel even wennen 😄) maar dan stel ik voor om een nederlands woord voor _Editorial_ te gebruiken (_Readactioneel_ klinkt niet heel best..). Anderen nog suggesties?

Reply:
We kunnen inderdaad ook commits in het Engels doen. Dat is alleen niet wat we historisch hebben gedaan. Zeker voor documenten die al Engels zijn is dit waarschijnlijk waardevol. Ik moet zelf ook erg wennen aan commit messages in het Nederlands, maar heb er geen sterke mening over. Zolang het maar consistent is met de rest en het document in kwestie.

Voor Engels kunnen we dan inderdaad `Editorial` gebruiken. Voor Nederlands anders `Documentbeheer`?

Reply:
Ik heb er `Beheer:` van gemaakt, dat klinkt kort en is een duidelijk begrip voor andere lezers.